From: Gabriel Sanderson <gsanders@genome.wustl.edu>
Date: Wed, 23 Jan 2013 17:32:25 -0600
Subject: [PATCH] cleanup cppcheck xml

---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index cb2499a..b7b7a8c 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -38,7 +38,7 @@ pindel:	$(SOURCES:.cpp=.o)
 
 clean:
 	$(RM) pindel pindel-debug $(OBJECTS) $(OBJECTS_DEBUG) $(CPPCHECKS) \
-	    *.gcov *.gcda *.gcno gmon.out *.d *.o
+	    *.gcov *.gcda *.gcno gmon.out *.d *.o cppcheck-result*.xml
 
 # Pattern rule to create normal .o files	
 %.o: %.cpp
-- 
