From: Gabriel Sanderson <gsanders@genome.wustl.edu>
Date: Fri, 25 Jan 2013 15:53:41 -0600
Subject: [PATCH] Added distclean

---
 Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c58aa7d..01c4baf 100644
--- a/Makefile
+++ b/Makefile
@@ -19,6 +19,8 @@ cppcheck: Makefile.local
 clean:
 	make -C src clean
 
+distclean: clean
+
 Makefile.local:
 	@echo '# Local configuration' > $@
 	@echo '# Location of SAMTools' >> $@
@@ -41,4 +43,4 @@ Makefile.local:
 .PHONY: default all clean test pindel pindel-debug cppcheck acceptance-tests \
 	coverage-tests functional-tests regression-tests
 .PRECIOUS: Makefile.local
-.IGNORE: clean
+.IGNORE: clean distclean
-- 
